packages/react-instantsearch-core/src/connectors/connectNumericMenu.js

```diff
@@ -157,7 +156,7 @@ export default createConnector({
     if (!items.some(item => item.value === '')) {
       items.push({
         value: '',
-        isRefined: isEmpty(refinedItem),
+        isRefined: refinedItem && refinedItem.length === 0,
```

Comment:
Actually is it really an array? The return value of `find` is either `T | undefined`.

Reply:
very good point, need to investigate this